constellations/gemini-admin/src/utils/textInput.sass

```diff
@@ -0,0 +1,43 @@
+@import "../_.constants.sass"
+
+#text-input-wrapper
+    padding: $spacing1x 0
+
+    h2
+        font-size: $fontSize2
+
```

Comment:
I'd like to also differentiate the description from the label.
So can you give the description a color of `colorGrayMed`?